l10n_eu_product_adr_dangerous_goods/tests/test_product_product.py

```diff
@@ -0,0 +1,166 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestProductProduct(TransactionCase):
+    def setUp(self):
```

Comment:
can you please use `BaseCommon` as base class and `setUpClass` instead of `setUp`?